src/sagas/wallet.js

```diff
@@ -279,22 +280,26 @@ export function* loadTokens() {
 
   const wallet = yield select((state) => state.wallet);
 
-  const registeredTokens = yield getRegisteredTokens(wallet, true);
+  const registeredTokens = yield getRegisteredTokens(wallet);
+
+  yield put(setTokens(registeredTokens));
```

Comment:
When loading a new wallet we need to update the registered tokens on the state so they will show on the dashboard